integrationtest/hibernate-validator/build.gradle

```diff
@@ -0,0 +1,131 @@
+plugins{
```

Comment:
@siordache I know it's noob question, but how do I run this build from the root dir?